### ISSUE TITLE
Cannot disable "Automatic weekly updates" via menu

### ISSUE THREAD

**Issue by ghost** [alberti42/Volume-Control#9]
The menu option is checked, but selecting it doesn't change anything - at least visibility wise.

<img width="255" alt="2021-10-04_12-35-14" src="https://user-images.githubusercontent.com/7449997/135836890-b74b3c73-2fc2-4380-91c8-92fdaf8e966d.png">

**Comment by alberti42** [comment]
Dear Simon,

I understand that you refer to the "Automatic weekly updates". With this option, the program checks in the background (so nothing visible) once per week if an update is available. So, I would say that the behavior you experience is normal. Thanks for the question!

Best regards
Andrea

**Comment by slabber** [comment]
But how do you disable this option?  The UI seems to suggest that selecting the menu option will remove the "check" and then it won't automatically check for weekly updates.

For me, selecting this menu item does nothing.

**Comment by alberti42** [comment]
You are right. This is a bug. I will have a look into it. Thank you for pointing out the wrong behavior.

Andrea
